Podcast/SearchTableViewController.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  SearchTableViewController.swift
+//  Podcast
+//
+//  Created by Kevin Greer on 3/3/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+enum SearchType {
+    case episodes
+    case series
+    case people
+    case tags
+}
+
+protocol SearchTableViewControllerDelegate {
+    func searchTableViewController(controller: SearchTableViewController, didTapSearchResultOfType searchType: SearchType, model: Any)
```

Comment:
These functions that cause the cells to store the models and pass them around are because I'm trying to send the model (series, episode, or user) all the way back to the DiscoverViewController so it can present the correct detail view. I could do this by sending around indexes, and then the model only from the TabbedPageVC to the discoverVC instead if thats better